File: JAVA/2177. Find Three Consecutive Integers That Sum to a Given Number.java
class Solution {
    public long[] sumOfThree(long n) {
        if(n%3!=0)
        return new long[0];
        long[] a=new long[3];
        a[1]=n/3;
        a[0]=(n/3)-1;
        a[2]=(n/3)+1;
        return a;
    }
}

File: C++/2177. Find Three Consecutive Integers That Sum to a Given Number.cpp
class Solution {
public:
    vector<long long> sumOfThree(long long n) {
        vector<long long>a;
        if(n%3!=0)
        return a;
        a.push_back((n/3)-1);
        a.push_back((n/3));
        a.push_back((n/3)+1);
        return a;
    }
};